bedrock/firefox/templates/firefox/whatsnew/includes/fx104/tcp.html

```diff
@@ -0,0 +1,10 @@
7 1.27 0 0 0 1.269 1.273ZM619.501 265.604c.701 0 1.27-.57 1.27-1.273s-.569-1.273-1.27-1.273a1.27 1.27 0 0 0-1.269 1.273 1.27 1.27 0 0 0 1.269 1.273ZM625.644 281.784a1.27 1.27 0 0 0 1.269-1.273 1.27 1.27 0 1 0-2.538 0 1.27 1.27 0 0 0 1.269 1.273ZM624.944 297.607a1.27 1.27 0 0 0 1.269-1.272 1.27 1.27 0 0 0-1.269-1.273 1.27 1.27 0 0 0-1.269 1.273 1.27 1.27 0 0 0 1.269 1.272ZM615.683 312.646c.701 0 1.27-.57 1.27-1.273s-.569-1.273-1.27-1.273a1.27 1.27 0 0 0-1.269 1.273 1.27 1.27 0 0 0 1.269 1.273ZM600.049 323.027a1.27 1.27 0 0 0 1.269-1.273 1.27 1.27 0 0 0-1.269-1.273 1.27 1.27 0 0 0-1.269 1.273 1.27 1.27 0 0 0 1.269 1.273ZM580.367 327.633a1.27 1.27 0 0 0 1.269-1.273 1.27 1.27 0 0 0-1.269-1.273 1.27 1.27 0 0 0-1.269 1.273 1.27 1.27 0 0 0 1.269 1.273ZM559.614 325.846a1.272 1.272 0 0 0 0-2.545 1.27 1.27 0 0 0-1.269 1.272 1.27 1.27 0 0 0 1.269 1.273ZM540.967 316.936a1.27 1.27 0 0 0 1.269-1.273 1.27 1.27 0 1 0-2.539 0c0 .703.569 1.273 1.27 1.273ZM526.424 302.809c.701 0 1.27-.57 1.27-1.273s-.569-1.273-1.27-1.273a1.27 1.27 0 0 0-1.269 1.273 1.27 1.27 0 0 0 1.269 1.273ZM520.543 286.928a1.27 1.27 0 0 0 1.269-1.273 1.27 1.27 0 0 0-1.269-1.273 1.27 1.27 0 0 0-1.269 1.273 1.27 1.27 0 0 0 1.269 1.273ZM523.201 270.643a1.272 1.272 0 0 0 0-2.545 1.27 1.27 0 0 0-1.269 1.272 1.27 1.27 0 0 0 1.269 1.273ZM532.175 257.371c.701 0 1.27-.57 1.27-1.273s-.569-1.273-1.27-1.273a1.27 1.27 0 0 0-1.269 1.273 1.27 1.27 0 0 0 1.269 1.273ZM546.979 246.715a1.27 1.27 0 0 0 1.269-1.273 1.27 1.27 0 0 0-1.269-1.273c-.701 0-1.27.57-1.27 1.273s.569 1.273 1.27 1.273ZM570.667 263.04a1.27 1.27 0 0 0 1.269-1.272 1.27 1.27 0 0 0-1.269-1.273 1.27 1.27 0 0 0-1.269 1.273 1.27 1.27 0 0 0 1.269 1.272ZM574.548 283.811c.701 0 1.27-.57 1.27-1.273s-.569-1.273-1.27-1.273a1.27 1.27 0 0 0-1.269 1.273 1.27 1.27 0 0 0 1.269 1.273ZM578.863 308.042a1.27 1.27 0 0 0 1.269-1.273 1.27 1.27 0 0 0-1.269-1.273c-.701 0-1.27.57-1.27 1.273s.569 1.273 1.27 1.273ZM546.504 266.069a1.27 1.27 0 0 0 1.269-1.273 1.27 1.27 0 0 0-1.269-1.273c-.701 0-1.27.57-1.27 1.273s.569 1.273 1.27 1.273ZM545.182 285.947a1.27 1.27 0 0 0 1.269-1.273 1.27 1.27 0 0 0-1.269-1.273 1.27 1.27 0 0 0-1.269 1.273 1.27 1.27 0 0 0 1.269 1.273ZM559.019 302.851c.701 0 1.27-.57 1.27-1.273s-.569-1.273-1.27-1.273a1.27 1.27 0 0 0-1.269 1.273 1.27 1.27 0 0 0 1.269 1.273ZM600.895 294.601a1.27 1.27 0 0 0 1.269-1.273 1.27 1.27 0 0 0-1.269-1.273 1.27 1.27 0 0 0-1.269 1.273 1.27 1.27 0 0 0 1.269 1.273ZM594.825 273.832c.701 0 1.27-.57 1.27-1.273s-.569-1.273-1.27-1.273a1.27 1.27 0 0 0-1.269 1.273 1.27 1.27 0 0 0 1.269 1.273Z" fill="#333336"/><defs><clipPath id="b"><path fill="#fff" transform="translate(634 156)" d="M0 0h15.714v15.714H0z"/></clipPath><clipPath id="c"><path fill="#fff" transform="translate(428 156)" d="M0 0h15.714v15.714H0z"/></clipPath><clipPath id="d"><path fill="#fff" transform="translate(221 156)" d="M0 0h15.714v15.714H0z"/></clipPath><clipPath id="e"><path fill="#fff" transform="translate(75 241)" d="M0 0h146.294v118.281H0z"/></clipPath></defs></svg>
```

Comment:
**Question**: wondering why this svg is in an html file as an inline element rather than as a standalone svg?

Also wondering what `tcp` means? (more curious than anything)

Reply:
tcp = Total Cookie Protection. 

This is reused from a previous page that Alex built (https://github.com/mozilla/bedrock/pull/11914) so I'm not sure what the reason was at the time... maybe it originally needed to change colors in dark mode but then the design evolved along the way and the final image no longer changes colors. It at least saves an HTTP request and might load a bit faster than an external resource.